### PR TITLE
NAS-110927 / 21.08 / No UI option to change the readonly state of a ZVOL

### PR DESCRIPTION
**Pull request by AlexKarpov98**

### CODE REVIEW

File: src/app/pages/storage/volumes/zvol/zvol-form/zvol-form.component.ts [hunk 1]
```diff
@@ -238,6 +240,16 @@ export class ZvolFormComponent implements FormConfiguration {
         tooltip: helptext.zvol_sparse_tooltip,
         isHidden: false,
       },
+      {
+        type: 'select',
+        name: 'readonly',
+        placeholder: helptext.zvol_readonly_placeholder,
+        tooltip: helptext.zvol_readonly_tooltip,
+        options: [
+          { label: 'On', value: 'ON' },
```
Comment (denysbutenko) [inline src/app/pages/storage/volumes/zvol/zvol-form/zvol-form.component.ts:249]: non-translatable labels

**Review by undsoft — CHANGES_REQUESTED** [review]
The dropdown works correctly, however this PR seems to make another existing issue worse.
When user presses "Edit Zvol", the form is not populated with current values. This also happens on master, but at least on master it works on second attempt for some reason.
Please take a look at this as well.

File: src/app/pages/storage/volumes/zvol/zvol-form/zvol-form.component.ts (hunk 1 above)
Comment (AlexKarpov98) [reply src/app/pages/storage/volumes/zvol/zvol-form/zvol-form.component.ts]: Fixed

**Comment by AlexKarpov98** [comment]
@undsoft Fixed

**Review by undsoft — APPROVED** [review]
Looks good, thanks!

**Pull request merged**